### PR TITLE
Added conditional normalization.

### PR DESCRIPTION
**Pull request by MylesBartlett**

### CODE REVIEW

File: models/layers.py [hunk 1]
```diff
@@ -13,17 +13,22 @@ def forward(self, x):
         return x
 
 
-def norm(num_norm_groups, num_channels):
-    if num_norm_groups > 0:
-        return nn.GroupNorm(num_groups=num_norm_groups, num_channels=num_channels)
+def norm(num_channels, num_norm_groups, num_classes=0):
```
Comment (Stoffle) [inline models/layers.py:16]: The positional arguments have swapped order here, are you *certain* that's also been done *everywhere* it's called? In general it would be best to leave the order the same

Comment (MylesBartlett) [reply models/layers.py:16]: I double checked it. I swapped the order to make it consistent with nn.BatchNorm2d.

Comment (tmke8) [reply models/layers.py:16]: We could force named arguments for this function with `def norm(*, num_channels=1, num_norm_groups=0, num_classes=0)`

Comment (tmke8) [reply models/layers.py:16]: Ah didn't see Myles' reply.
